envoy/tests/test_envoy.py

```diff
@@ -36,18 +36,18 @@ def test_success(aggregator):
 
 
 @pytest.mark.unit
-def test_success_fixture(aggregator):
+def test_success_fixture(aggregator, fixture_path, mock_http_response):
     instance = INSTANCES['main']
     c = Envoy(CHECK_NAME, {}, [instance])
 
-    with mock.patch('requests.get', return_value=response('multiple_services')):
-        c.check(instance)
+    response = mock_http_response(file_path=fixture_path('multiple_services')).return_value
```

Comment:
Is this `response` used anywhere?

Reply:
https://github.com/DataDog/integrations-core/pull/10242

Reply:
Yes a few lines down